docs/contributors_guide/maintenance/simp_puppet_module_release_procedures/Pre_Release_Checklist.rst

```diff
@@ -0,0 +1,301 @@
+
+* Verify there are no dependencies in the ``metadata.json`` file
+  that do not exist in the ``build/rpm_metadata/requires`` and
+  vice versa, except for ``puppetlabs/stdlib`` and ``simp/simplib``.
+
+* Verify that the version constraints for each dependency are
+  the same in both files.
+
+.. IMPORTANT::
+
+   The ``puppetlabs/stdlib`` and ``simp/simplib`` dependencies are
+   automatically added to the compenent's RPM requires list, when the
+   RPM is built.  So, these dependencies **only** have to be listed
+   in the ``build/rpm_metadata/requires`` file, **if** the version(s)
+   the module requires are newer than than those specified in the
+   `RPM spec file template`_ .
+
+Verify a Puppet module can be created
+-------------------------------------
+
+This check verifies that a `PuppetForge`_-deployable Puppet module can
+be created:
+
+.. code-block:: bash
+
+   bundle exec rake metadata_lint
+   puppet module build
+
+Verify RPMs can be created
+--------------------------
+
+This check verifies that CentOS 6 and CentOS 7 RPMs can be generated
+for this module:
+
+.. code-block:: bash
+
+   bundle exec rake pkg:rpm[epel-6-x86_64]
+   bundle exec rake pkg:rpm[epel-7-x86_64]
+
+Verify unit tests pass
+----------------------
+
+This check verifies that the component's unit tests have succeeded
+in `TravisCI`_:
+
+* Navigate to the project's TravisCI results page and verify the
+  tests for the development branch to be tagged and released have
+  passed.  For our project, this page is
+  https://travis-ci.org/simp/pupmod-simp-iptables/branches
+
+.. IMPORTANT::
+
+   If the tests in TravisCI fail, you **must** fix them before
+   proceeding.  The automated release procedures will only
+   succeed, if the unit tests succeed in TravisCI.
+
+Verify acceptance tests pass
+----------------------------
+
+This check verifies that the component's acceptance tests have 
+succeeded:
+
+* Run the ``beaker:suites`` rake task with and without FIPS enabled
+
+  .. code-block:: bash
+
+     BEAKER_fips=yes bundle exec rake beaker:suites
+     bundle exec rake beaker:suites
+
+.. NOTE::
+
+   * For older projects that have not been updated to use test
+     suites, you may have to run the ``acceptance`` rake task,
+     instead.
+
+   * If the GitLab instance for the project is current (it is
+     sync'd every 3 hours), you can look at the latest acceptance
+     test results run by GitLab.  For our project, the results will
+     be at https://gitlab.com/simp/pupmod-simp-iptables/pipelines.
+
+Verify interoperability with last SIMP release
+----------------------------------------------
+
+This check verifies that this version of the component interoperates
+with the last full SIMP release. For many components, the best
+automated way of doing this is by running the ``pupmod-simp-simp``
+acceptance tests, as these tests provide extensive, multi-component,
+integration tests.
+
+#. Determine the version of ``pupmod-simp-simp`` used in the last SIMP
```

Comment:
The default acceptance test in simp-core is also good. You'd have to drop in your own Puppetfile, though